docs/diagrams/StorageClassDiagram.puml

```diff
@@ -29,15 +29,16 @@ HiddenOutside ..> Storage
 
 StorageManager .up.|> Storage
 StorageManager -up-> "1" UserPrefsStorage
-StorageManager -up-> "1" AddressBookStorage
+StorageManager -up-> "1" TaAssistStorage
 
 Storage -left-|> UserPrefsStorage
-Storage -right-|> AddressBookStorage
+Storage -right-|> TaAssistStorage
 
 JsonUserPrefsStorage .up.|> UserPrefsStorage
-JsonAddressBookStorage .up.|> AddressBookStorage
-JsonAddressBookStorage ..> JsonSerializableAddressBook
-JsonSerializableAddressBook --> "*" JsonAdaptedStudent
-JsonAdaptedStudent --> "*" JsonAdaptedTag
+JsonTaAssistStorage .up.|> TaAssistStorage
+JsonTaAssistStorage ..> JsonSerializableTaAssist
+JsonSerializableTaAssist--> "*" JsonAdaptedStudent
+JsonSerializableTaAssist--> "*" JsonAdaptedModuleClass
+JsonAdaptedStudent --> "*" JsonAdaptedModuleClass
```

Comment:
```suggestion
```

In PR #114, this is not the case. `JsonAdaptedStudent` actually stores a list of `String`-s of the names of the `ModuleClass`-es. Whilst not the most intuitive design, this allows the `json` file to be more compact with less redundant information.